ports/coreMQTT/azure_iot_core_mqtt.c

```diff
@@ -190,7 +190,7 @@ uint16_t AzureIoTMQTT_GetPacketId( AzureIoTMQTTHandle_t xContext )
 
     xResult = MQTT_GetPacketId( xContext );
 
-    return prvTranslateToAzureIoTMQTTResult( xResult );
+    return ( uint16_t ) prvTranslateToAzureIoTMQTTResult( xResult );
```

Comment:
>prvTranslateToAzureIoTMQTTResult [](http://example.com/codeflow?start=24&length=32)

:( its a bug. Can you do following thing:

uint16_t AzureIoTMQTT_GetPacketId( AzureIoTMQTTHandle_t xContext )
{
    return  MQTT_GetPacketId( xContext );
}

Reply:
ahhh yup I should have checked. updated